zulip/integrations/trello/zulip_trello.py

```diff
@@ -9,7 +9,12 @@
 
 import sys
 import argparse
-import requests
+
+try:
+    import requests
+except ImportError:
+    print("Error: Please install the `requests` library before running this script")
```

Comment:
Maybe "Please install the python requests library before running this script: http://docs.python-requests.org/en/master/user/install/"